Content.Server/Ninja/Systems/NinjaSuitSystem.cs

```diff
@@ -0,0 +1,155 @@
+using Content.Server.Actions;
+using Content.Server.Emp;
+using Content.Server.Popups;
+using Content.Server.Power.Components;
+using Content.Server.PowerCell;
+using Content.Shared.Actions;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Ninja.Components;
+using Content.Shared.Ninja.Systems;
+using Content.Shared.Popups;
+using Robust.Shared.Containers;
+
+namespace Content.Server.Ninja.Systems;
+
+public sealed class NinjaSuitSystem : SharedNinjaSuitSystem
+{
+    [Dependency] private readonly ActionsSystem _actions = default!;
+    [Dependency] private readonly EmpSystem _emp = default!;
+    [Dependency] private readonly SharedHandsSystem _hands = default!;
+    [Dependency] private readonly new NinjaSystem _ninja = default!;
```

Comment:
```suggestion
    [Dependency] private readonly NinjaSystem _ninja = default!;
```
This shouldn't be new.

Reply:
oh yeah forgot i removed it from shared